Automattic/jetpack
# Instant Search: Update styles to reduce usage of "!important".

#### Changes proposed in this Pull Request:


Introduces some more specific CSS rules for the various styled buttons in the Overlay. Separated out and limited to those styles (font/color attributes) that are currently dependant on `!important` directives. Necessary to prevent aggressive theme styles from bleeding into the Overlay.

With this technique, it would still be possible for site owners to override the styles as needed.

Also updates the font weights to match what we had initially before the Twenty Twenty One issues surfaced.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit a site with a Jetpack Search plan.
2. Make sure the Twenty Twenty One theme is active. It has the aggressive theme styles we are trying to address.
3. Open the Overlay by appending `?s=&category=uncategorized` to your site address.
4. Confirm the Sort Options buttons look correct.
5. Confirm the "Clear filters" button looks correct.
6. Clear the filters and confirm the "Load more" button looks correct.
7. Inspect the various buttons and confirm there are no `!important` attributes used for size/color.

You can toggle the light/dark themes from `/wp-admin/admin.php?page=jetpack-search-configure`.
